tests/test_examples.py

```diff
@@ -57,7 +57,12 @@ def init_class_fixtures(cls):
 
         market_data = ('SPY_benchmark.csv', 'treasury_curves.csv')
         for data in market_data:
-            ensure_file(cls.tmpdir.getpath('example_data/root/data/' + data))
+            update_modified_time(
+                cls.tmpdir.getpath(
+                    'example_data/root/data/' + data
+                ),
+                None
```

Comment:
You were asking yesterday about the default that was added in python 3. You could abstract that away in our code by making `None` the default value of the `times` argument to `update_modified_time`.